study.md

```diff
@@ -39,7 +73,10 @@ sum = left_operand.to_i + right_operand.to_i
 Write some JavaScript code that duplicates this functionality.
 
 ```javascript
-// your answer here
+let left_operand = '1'
+let right_operand = '2'
+
+let sum = left_operand.parseFloat + right_operand.parseFloat
```

Comment:
This will result in `NaN` since invoking `parseFloat` on a string will return `undefined`.